examples/configuring/README.md

```diff
@@ -0,0 +1,65 @@
+Configuring GraphDB
+===
+
```

Comment:
Add some paragraph explaining what this readme is about as well each example purpose...